docs/config/README.md

```diff
@@ -713,6 +713,7 @@ To enable it, set `disabled` to `false` in your configuration file.
 | `show_swap`       | `true`                | Display swap usage if total swap is non-zero.                 |
 | `threshold`       | `75`                  | Hide the memory usage unless it exceeds this percentage.      |
 | `symbol`          | `"🐏 "`               | The symbol used before displaying the memory usage.           |
+| `seperator`       | `" | "`               | The symbol or text that will seperate the ram and swap usage. |
```

Comment:
Isn't it written `separator`? https://duckduckgo.com/?q=seperator 😃

Reply:
wow, can't believe I spelled it wrong there and in the actual module code! Good catch :)